### PR TITLE
add changelogs for 3.0.6-3.0.8, fix user agent

### PR DESCRIPTION
**Pull request by aaronm67**
Versioning fixes
- Reflect v3.0.8 in the client's user agent (3.0.8 will be the version this is released under)
- Add changelogs for 3.0.5-3.0.8

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,16 @@
 # Changelog
+## [3.0.8] - 2019-12-11
```
Comment (tommypenner) [inline CHANGELOG.md:2]:
```suggestion
## [3.0.8] - 2021-05-03
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,16 @@
...
+### Fixed
+- Fix user-agent in vimeo.php
+
+## [3.0.7] - 2019-12-11
```
Comment (tommypenner) [inline CHANGELOG.md:6]:
```suggestion
## [3.0.7] - 2021-04-28
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,16 @@
...
+### Fixed
+- Fix an issue with SSL certificates on certain requests ([#286](https://github.com/vimeo/vimeo.php/issues/286))
+
+## [3.0.6] - 2019-12-11
```
Comment (tommypenner) [inline CHANGELOG.md:10]:
```suggestion
## [3.0.6] - 2020-06-20
```

**Review by tommypenner — CHANGES_REQUESTED** [review]
Fixed dates

File: CHANGELOG.md (hunk 1 above)
Comment (aaronm67) [reply CHANGELOG.md]: Oops, thanks - I didn't even  notice there were dates on there.

**Review by Sventour — APPROVED** [review]

**Review by nbeliard — APPROVED** [review]
LGTM

**Review by tommypenner — APPROVED** [review]

**Pull request merged**